src/java.base/share/classes/java/lang/Process.java

```diff
@@ -149,12 +169,107 @@ public abstract class Process {
     private Charset inputCharset;
     private BufferedReader errorReader;
     private Charset errorCharset;
+    private boolean closed;     // true if close() has been called
 
     /**
      * Default constructor for Process.
      */
     public Process() {}
 
+    /**
+     * Close all writer and reader streams immediately and wait for the process to terminate.
```

Comment:
It might read a bit better to say "reader and write streams" and drop the word immediately.